modules/ROOT/pages/_partials/keycloak/8_setting-up-the-idm-sdk.adoc

```diff
@@ -0,0 +1,234 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+[id='setting-up-sdk-{context}']
+= Setting up the {keycloak-service} Service SDK
+
+This section helps you to set up the {keycloak-service} service SDK in your App.
+It describes how to set up and initialize the {keycloak-service} service SDK.
+
+.Prerequisites
+
+* Your local `mobile-services.json` file is up-to-date.
+
+.Procedure
+
+1. Import the libraries
+
+[role="primary"]
+.Android
+****
+. Add the following dependency in your app's *build.gradle*:
++
+[source,groovy,subs="attributes"]
+----
+dependencies {
+    implementation "org.aerogear:android-auth:{release-number}"
+}
+----
+. To prevent build errors, add the following in your app's *build.gradle*:
+[source,groovy,subs="attributes"]
++
+----
+compileOptions {
+    sourceCompatibility 1.8
+    targetCompatibility 1.8
+}
+----
+****
+
+[role="secondary"]
+.iOS
+****
+. Add the dependency to your *Podfile*:
++
+[source,ruby,subs="attributes"]
+----
+target '[TARGET NAME]' do
+    pod 'AGSAuth', '{release-number}'
+end
+----
+
+. Update the dependencies:
++
+[source,bash]
+----
+$ pod install
+----
+
+. Import and instantiate `AGSAuth` to start using the SDK:
++
+[source,swift]
+----
+import AGSAuth
+
+auth = AGSAuth()
+----
+****
+
+[role="secondary"]
+.Cordova
+****
+Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM, window="_blank"]:
+[source,bash]
+----
+$ npm install @aerogear/auth
```

Comment:
@finp - fyi: results of this command are:

[jstaffor@jstaffor test_cordova_project]$ npm install @aerogear/auth
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN helloworld@1.0.0 No repository field.

+ @aerogear/auth@2.0.0
added 14 packages from 23 contributors and audited 15 packages in 5.361s
found 0 vulnerabilities